### ISSUE TITLE
ui5-responsive-popover: position of close icon in phone mode is not correct

### ISSUE THREAD

**Issue by zhangliangyi** [SAP/ui5-webcomponents#3475]
### **Bug Description**
It is expected that the `ui5-responsive-popover` is opened with a dialog in phone device. But the close icon of the dialog is not put in the right top corner of dialog, but wrapped to the left side.

### **Expected Behavior**
The close icon should be put in the right top corner of the dialog header when opened in phone device. Here is the screenshot from the `ResponsivePopover` in sapui5.
<img width="300" alt="2021-06-25_17-18-39" src="https://user-images.githubusercontent.com/1867889/123401629-8d4a2c00-d5d9-11eb-93c5-27773085a6b9.png">

### **Steps to Reproduce**
1. Go to https://sap.github.io/ui5-webcomponents/playground/components/ResponsivePopover/
2. Toggle the device emulation of browser to iphone X
3. Refresh the page
4. Click the button in first example
5. The dialog opened would be like:
<img width="300" alt="2021-06-25_17-22-04" src="https://user-images.githubusercontent.com/1867889/123402130-0184cf80-d5da-11eb-9c64-fb70094a63d3.png">

### **Context**
- UI5 Web Components version: 0.31.11
- OS/Platform: MacOS
- Browser: Edge
- Affected component: ui5-responsive-popover

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

- Organization: SF

**Comment by NHristov-sap** [comment]
Hi @SAP/ui5-webcomponents-topic-rd,

The issue is reproducible, see details above. A brief look at the DOM shows that possible reason is a bug in class asignment:

```
<ui5-dialog stretch="" _disable-initial-focus="" ui5-dialog="" on-phone="" opened="" style="top: 0px; left: 0px; z-index: 104; display: flex;">
#shadow-root (open)
<header class="[object Object]">
----------------------^
...
</ui5-dialog>
```

Best Regards,
Nikolay Hristov

**Comment by gmkv** [comment]
This is already fixed by #3176
@ilhan007 do you know when the fix will be available?

**Comment by gmkv** [comment]
Hi @zhangliangyi 
This issue is fixed by #3176 
It will be avaliable with the next release of ui5-webcomponents.

You can see the fix working in the master branch:
https://sap.github.io/ui5-webcomponents/master/playground/components/ResponsivePopover/

Thanks for reporting!
Kind regards,
Georgi

**Comment by zhangliangyi** [comment]
@gmkv @ilhan007 
Please don't forget to add this fix into next 0.31.x release, thanks.
